packages/story-editor/src/components/footer/footerLayout.js

```diff
@@ -52,17 +53,19 @@ const Area = styled.div`
 
 function FooterLayout({ footer }) {
   return (
-    <Wrapper aria-label={__('Workspace Footer', 'web-stories')}>
-      <Area area="carousel">
-        <Carousel />
-      </Area>
-      <Area area="primary">
-        <PrimaryMenu />
-      </Area>
-      <Area area="secondary">
-        <SecondaryMenu menu={footer?.secondaryMenu} />
-      </Area>
-    </Wrapper>
+    <DirectionAware>
```

Comment:
Added `<DirectionAware>` to make spacing consistent in RTL, otherwise the layers panel toggle is right up against the edge of the browser